project/src/main/webapp/events-feed.html

```diff
@@ -45,6 +45,10 @@ <h3>Discover Events Around You</h3>
           aria-expanded="false"
         >
           Filter
+        <span class="caret"></span></button>
+        <ul class="dropdown-menu" id="filterCheckBoxes" role="menu" aria-labelledby="menu1">
+        <button class="filter-btn" type="submit" onclick="getFilteredEvents()">Filter</button>
```

Comment:
nit: indent this button since it's inside the list

Reply:
done